include/triton/Conversion/TritonGPUToLLVM/Utility.h

```diff
@@ -1058,20 +1060,23 @@ DenseMap<unsigned, Value> static getSwizzledSharedPtrs(
   // cache for non-immediate offsets
   DenseMap<unsigned, Value> cacheCol, cacheRow;
   unsigned minVec = std::min(outVec, inVec);
+  Value strideRow = outOrder.size() == 2 ? srcStrides[outOrder[1]] : i32_val(0);
```

Comment:
should this be `outOrder.size() >= 2` ? ...

Reply:
Yes you are right! It is supposed to pull the logic out for strideRow
    if (outOrder.size() >= 2) {
      strideRow = srcStrides[outOrder[1]];
    } else {
      strideRow = i32_val(0);
    }